applications/projects/runSofa/Main.cpp

```diff
@@ -374,8 +374,8 @@ int main(int argc, char** argv)
     msg_info(appName) << "GuiDataRepository paths = " << GuiDataRepository.getPathsJoined();
 
     // Initialise paths
-    BaseGUI::setConfigDirectoryPath(Utils::getSofaPathPrefix() + "/config", true);
-    BaseGUI::setScreenshotDirectoryPath(Utils::getSofaPathPrefix() + "/screenshots", true);
+    BaseGUI::setConfigDirectoryPath(Utils::getSofaUserLocalDirectory() + "/config", true);
+    BaseGUI::setScreenshotDirectoryPath(Utils::getSofaDataDirectory() + "/screenshots", true);
```

Comment:
```suggestion
    BaseGUI::setConfigDirectoryPath(FileSystem::append(Utils::getSofaUserLocalDirectory(), "config"), true);
    BaseGUI::setScreenshotDirectoryPath(FileSystem::append(Utils::getSofaDataDirectory(), "screenshots"), true);
```
to ensure exactly one directory separator (/).